util/conversion/conversion.go

```diff
@@ -50,3 +62,28 @@ func UnmarshalData(from metav1.Object, to interface{}) (bool, error) {
 	delete(from.GetAnnotations(), DataAnnotation)
 	return true, nil
 }
+
+// GetFuzzer returns a new fuzzer to be used for testing.
```

Comment:
Shouldn't these belong in a _test.go file?

Reply:
I don't think I can import them if they're in a test file

Reply:
I can try though, maybe I'm misremembering

Reply:
yeah confirmed, it fails with

```
/Users/vince/go/src/sigs.k8s.io/cluster-api/api/v1alpha2/conversion_test.go:37:23: undefined: "sigs.k8s.io/cluster-api/util/conversion".FuzzTestFunc
FAIL	sigs.k8s.io/cluster-api/api/v1alpha2 [build failed]
```
if it's part of the test files